VCPkg does not support the v143 MSVC toolset

**Describe the bug**
VCPkg does not support the v143 MSVC toolset. It seems to stem from a missing entry in this hard-coded list:
https://github.com/microsoft/vcpkg-tool/blob/3be6b75f6972aa72ca34afce39168094432ea300/src/vcpkg/visualstudio.cpp#L15-L18
And subsequently a missing case here:
https://github.com/microsoft/vcpkg-tool/blob/main/src/vcpkg/visualstudio.cpp#L252-L266

**Environment**
- OS: Windows
- Compiler: VS 2022 17.0 Preview 2.0

**To Reproduce**
Given some valid cmake project:
1. `cmake -B build -S . -G "Visual Studio 17 2022" -T v143 -DCMAKE_TOOLCHAIN_FILE="path/to/vcpkg/scripts/buildsystems/vcpkg.cmake"`
2. See error
```
-- Running vcpkg install
No suitable Visual Studio instances were found
-- Running vcpkg install - failed
CMake Error at E:/Repos/vcpkg/scripts/buildsystems/vcpkg.cmake:837 (message):
  vcpkg install failed.  See logs for more information:
  E:\Repos\bsa\build\vcpkg-manifest-install.log
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.21/Modules/CMakeDetermineSystem.cmake:124 (include)
  CMakeLists.txt:3 (project)
```

**Expected behavior**
I would expect vcpkg to recognize the v143 toolset.

**Failure logs** 
`vcpkg-manifest-install.log`
```
No suitable Visual Studio instances were found
```

--- a/scripts/bootstrap.ps1
+++ b/scripts/bootstrap.ps1
@@ -45,7 +45,7 @@ while (!($vcpkgRootDir -eq "") -and !(Test-Path "$vcpkgRootDir\.vcpkg-root"))
 
 Write-Verbose "Examining $vcpkgRootDir for .vcpkg-root - Found"
 
-& "$scriptsDir/tls12-download.exe" github.com "/microsoft/vcpkg-tool/releases/download/2021-07-21/vcpkg.exe" "$vcpkgRootDir\vcpkg.exe"
+& "$scriptsDir/tls12-download.exe" github.com "/microsoft/vcpkg-tool/releases/download/2021-07-26/vcpkg.exe" "$vcpkgRootDir\vcpkg.exe"
 Write-Host ""
 
 if ($LASTEXITCODE -ne 0)

--- a/scripts/bootstrap.sh
+++ b/scripts/bootstrap.sh
@@ -278,8 +278,8 @@ else
 fi
 
 # Do the build
-vcpkgToolReleaseTag="2021-07-21"
-vcpkgToolReleaseSha="9d1644c1c34bfd750cad36c47d7997698b0c5c4ff0673f9f39ff5f62f6e351ca32db7991043e7f154aa8ffe8b5180591d78131edf5dc533164d75a60764a8da8"
+vcpkgToolReleaseTag="2021-07-26"
+vcpkgToolReleaseSha="e5252df535a75e1b776e057811a2913b0508d0a387a781815e353c23d0a33ec8bf8982a121adc6b711243bf0981843a435a0e53facf274ac7c38c3f571c26766"
 vcpkgToolReleaseTarball="$vcpkgToolReleaseTag.tar.gz"
 vcpkgToolUrl="https://github.com/microsoft/vcpkg-tool/archive/$vcpkgToolReleaseTarball"
 baseBuildDir="$vcpkgRootDir/buildtrees/_vcpkg"

